apps/aot/src/components/navigation/logo.tsx

```diff
@@ -1,6 +1,12 @@
 export default function Logo() {
   return (
-    <svg className="h-6" viewBox="0 0 569 123" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <svg
+      aria-label="advent<TS>"
```

Comment:
When an `<svg>` or other visual asset doesn't have a label or title, there's no way  to know what it is programmatically. This is bad both for screenreaders and for SEO.

https://web.dev/learn/accessibility/images/